cli/ballerina-cli-utils/src/main/ballerina/packaging_search/packaging_search.bal

```diff
@@ -33,7 +33,7 @@ function search (http:Client definedEndpoint, string url, string querySearched,
     } else {
         jsonResponse = check (httpResponse.getJsonPayload());
         json[] artifacts = check <json[]> jsonResponse.artifacts;
-        if (artifacts == null || lengthof artifacts > 0) {
+        if (lengthof artifacts > 0) {
```

Comment:
What does this have to do with equality check?

Reply:
Here artifacts can never be null. If there are no packages to be found we simply return an artifact array of zero. Since we already check if the artifact array size is greater than zero it is okay to remove this check

Reply:
Yeah, basically an array cannot be `null` (default is an empty array), but we had allowed this equality check previously. That equality check is no longer allowed, thus this change.